docs/tutorials/blogs.md

```diff
@@ -0,0 +1,1087 @@
+# Blog tutorial
+
+## 1. Overview
+__Content:__ The tutorial guides you through the process of configuring the
+[blog plugin], creating posts, setting up authors, providing secondary navigation
+with tags and producing an RSS feed.
+
+[blog plugin]: ../plugins/blog.md
+
+__Prerequisites:__ This tutorial assumes that you have installed either the
+[public version] or the [Insiders edition] of Material for MkDocs and that you have
+worked through the [Creating your site] setup guide. Note that where the features
+we use the Insiders edition, we mark these with the heart icon:
+<!-- md:sponsors --> If you are using the public version then you can skip these
+steps. Sometimes there will be ways of achieving the same goal that differ
+between the public version and the Insider edition. In that case, we will show
+them in a tabbed view so you can see one or the other.
+
+[public version]: ../getting-started.md
+[Insiders edition]: ../insiders/getting-started.md
+[Creating your site]: ../creating-your-site.md
+
+
+__Outcomes:__ After working through the tutorial, you will have learned how to
+produce a professional or personal blog to engage with your audience. You will
+know and have experienced how you can use the blog plugin in combination with
+other features and third-party plugins to achieve this.
+
+__Time required:__ To run through this tutorial in full you will typically need
+to have about XX minutes of time. Note that the time it takes depends on a
+number of factors, so this is only a rough estimation based on our experience
+and testing.
+
+## 2. Introduction
+
+Blogs are a great way to engage with your audience. Software developers can use
+a blog to announce new features, demonstrate their usage and provide background
+information. You can demonstrate competence by commenting on the state of the
+art or document your own work as best practice. Posts on current topics can help
+draw in visitors for your main website and can keep your audience engaged. Of
+course, you can blog about any topics close to your heart.
+
+The [blog plugin] makes running a blog alongside your other content easy but you
+can also configure it to run a stand-alone blog if posts are the only kind
+of content you need. It becomes even more powerful when used in combination with
+other features of Material for MkDocs such as the [tags plugin] or with third
+party plugins such as the [RSS plugin].
+
+[tags plugin]: ../../plugins/tags
+[RSS plugin]: https://guts.github.io/mkdocs-rss-plugin/
+
+!!! example "Explore the Material for MkDocs Blog"
```

Comment:
This sounds like a university assignment to me 😉

Reply:
You got me... I will get rid of this and just invite people to check out our blog once they have read about the elements of a blog. I also wonder if we should say that people can skip things they are familiar with.

Reply:
Yeah. And don't forget to mention the tests they have to pass with straight A's to be allowed to use our product ✌️